fe/fe-core/src/main/java/org/apache/doris/catalog/DataProperty.java

```diff
@@ -95,14 +138,20 @@ public boolean equals(Object obj) {
         DataProperty other = (DataProperty) obj;
 
         return this.storageMedium == other.storageMedium
-                && this.cooldownTimeMs == other.cooldownTimeMs;
+                && this.coolDownTimeMs == other.coolDownTimeMs
+                && this.remoteCoolDownTimeMs == other.remoteCoolDownTimeMs
+                && this.remoteStorageResourceName.equals(other.remoteStorageResourceName)
+                && this.remoteStorageMedium == other.remoteStorageMedium;
```

Comment:
when remote storage medium is used?

Reply:
The remote storage medium is designed to make it easier to get the redundant remote storage media without having to get it from the resource manager every time. 
It is mainly used to determine the storage media that may be needed several times during the process of converting hot data to cold data.